raptor/tree_builder.py

```diff
@@ -269,7 +269,8 @@ def build_from_text(self, text: str, use_multithreading: bool = True) -> Tree:
             Tree: The golden tree structure.
         """
         chunks = split_text(text, self.tokenizer, self.max_tokens)
-
+        chunks = [chunk for chunk in chunks if chunk.strip()]
```

Comment:
I feel that this should belong in the split_text function instead of the tree_builder. Can you move it there instead.